docs/pages/setup/reference/cli.mdx

```diff
@@ -1310,6 +1310,346 @@ $ tctl auth rotate --type=user --grace-period=200h
 $ tctl auth rotate --type=host --grace-period=8h
 ```
 
+### tctl sso test
+
+Perform an end-to-end test of an SSO authentication flow using the provided auth connector definition.
+
+The command supports all auth connector types: `github`, `oidc` and `saml`. The latter two require Teleport Enterprise.
+
+```code
+$ tctl [<global-flags>] sso test [<auth-connector.yaml>]
+```
+
+The testing consists of running a single end-to-end authentication request using the provided auth connector definition.
+Once the request is finished, the results will be printed to console along with context-specific diagnostic information.
```

Comment:
```suggestion
Once the request is finished, the results will be printed to standard output along with context-specific diagnostic information.
```
In case the user wants to do shell redirection